src/main/java/TestFile2.java

```diff
@@ -0,0 +1,16 @@
+class TestFile2 {
+    public String foo() {
+        return "bar";
+    }
+
+    private static void incorrectAssignmentInIfCondition() {
+        boolean value = false;
```

Comment:
[Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.